simon/_posts/2017-10-16-day-33.md

```diff
@@ -12,5 +12,8 @@ Things to write about:
 * How many books did you finish since your last journal entry? **Record the number** and write what they were.
 * Did you do any music or crafts?
 
-***
+I read creation and wrote about these puncuation marks (here are the puncation marks. ?!.). Then i corected my math and workedon my spelling . Then i made a sheduale.
```

Comment:
Did you write "about these punctuation marks?" Or write sentences using punctuation marks?

Check spelling for:

* punctuation
* corrected
* worked on
* schedule